packages/dds/tree/src/tableSchema.ts

```diff
@@ -182,7 +223,7 @@ export namespace TableSchema {
 	 */
 	export type ColumnSchemaBase<
 		TScope extends string | undefined,
-		TPropsSchema extends ImplicitFieldSchema,
+		TPropsSchema extends ImplicitFieldSchema = ImplicitFieldSchema,
```

Comment:
A bit of unrelated cleanup to make locations that reference `ColumnSchemaBase` a bit simpler.